src/__test__/pet.test.js

```diff
@@ -0,0 +1,183 @@
+const Pet = require('../../src/pet');
+
+describe('constructor', () => {
+    it('returns an object', () => {
+      expect(new Pet('Fido')).toBeInstanceOf(Object);
+    });
+  });
+
+  describe('constructor', () => {
+    it('sets the name property', () => {
+      const pet = new Pet('Fido');
+  
+      expect(pet.name).toEqual('Fido');
+    });
+  });
+
+  describe('constructor', () => {
+  
+    it('has a initial age of 0', () => {
+      const pet = new Pet('Fido');
+  
+      expect(pet.age).toEqual(0);
+    });
+  });
```

Comment:
You have quite a few describe blocks here that dont need to be, you should group all these together under a single describe block:
```
describe("constructor", () => {
    it("returns an object", () => {
        expect(new Pet("Fido")).toBeInstanceOf(Object);
    });

    it("sets the name property", () => {
        const pet = new Pet("Fido");

        expect(pet.name).toEqual("Fido");
    });

    it("has a initial age of 0", () => {
        const pet = new Pet("Fido");

        expect(pet.age).toEqual(0);
    });

    it("has a initial hunger of 0", () => {
        const pet = new Pet("Fido");

        expect(pet.hunger).toEqual(0);
    });

    it("has a initial fitness of 10", () => {
        const pet = new Pet("Fido");

        expect(pet.fitness).toEqual(10);
    });

    it("has an empty child array", () => {
        const pet = new Pet("Fido");
        expect(pet.children.length).toBe(0);
    });
});
```

You could do the same for all the other blocks too, eg 'grow up', 'check up' etc